src/index.ts

```diff
@@ -208,6 +253,102 @@ const main = (vortex: VortexExtensionContext) => {
     */
   }
 
+  vortex.registerReducer([`settings`, `redmod`], reducer);
+
+  vortex.registerSettings(`RedMods`, settings, undefined, () => {
+    const state = vortex.api.store.getState();
+    const gameMode = vortexApi.selectors.activeGameId(state);
+    return gameMode === GAME_ID;
+  }, 51);
+
+  // REDmod TODO
+  vortex.registerToDo(
```

Comment:
Todo is for prompting the user to do something at startup and/or dismiss, right? I have to actually try and see what these look like!

Looks like you're doing some work here already, is it possible/feasible to instead make the changes through the Settings view only in order to reduce possible code paths?

Reply:
After thinking on it this morning after playing with them last night, I realized that the TODOs aren't needed for each setting added. Ya, they almost look like sticky notes on the dashboard page on vortex (when you think back about them). In fact, we don't _have_ to add these at all. We could have it just be text that tells the user to checkout the new settings page under settings, for sure.

Reply:
Maybe (also related to the channel discussion) if we have some single "welcome to 0.9, do this" TODO or migration message?